### DISCUSSION TITLE
Is there a way to send selections to the quickfix list without opening the list afterwards?

### DISCUSSION THREAD

**Discussion by KodyVB** [ibhagwan/fzf-lua#1592] [Q&A] [answered]
When I send selections to the quickfix list, it closes the fzf-lua prompt as expected, but it also opens the quickfix list as a split, which I'd rather it not do. Is there a way to change this behavior so it just sends the selections to the list without visually opening it?

**Comment by ibhagwan** [comment, marked as answer]
I’m AFK so I can’t recall the exact option name, search  the issues it might be just adding `copen = false` to the command.

**Reply by KodyVB** [reply]
Perfect, it was settable via `defaults.copen = false`
Thanks for the quick response - I looked all over the place searching for quickfix but didn't think to search for copen.
